### PR TITLE
chore(docs): include information about using `suite` property in conf…

### PR DESCRIPTION
**Pull request by sjelin**
…ig file

### CODE REVIEW

**Comment by Droogans** [comment]
Hey there noticed this hasn't been merged after a couple weeks. I'd appreciate it in the docs.

**Comment by sjelin** [comment]
@juliemr @hankduan ?

File: docs/referenceConf.js [hunk 1]
```diff
@@ -99,6 +99,9 @@ exports.config = {
     smoke: 'spec/smoketests/*.js',
     full: 'spec/*.js'
   },
+  // If you would like protractor to use a specific suite by default instead of
+  // all suites, you can put that in the config file as well
```
Comment (juliemr) [inline docs/referenceConf.js]: Periods at the end of sentences in comments.

**Comment by juliemr** [comment]
Style nit, then LGTM

**Pull request merged**